src/Symfony/Component/HttpKernel/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ CHANGELOG
 3.1.0
 -----
  * deprecated passing objects as URI attributes to the ESI and SSI renderers
+ * Added a `LegacyArgumentResolver` with `getArguments()` and the respective interface `ArgumentResolverInterface`
```

Comment:
technically, the new feature is adding the interface. Your sentence makes it a side effect of adding the class